recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "autogluon.multimodal" %}
-{% set version = "1.1.1" %}
+{% set version = "0.8.3" %}
```

Comment:
This update will break the master as it is currently pointing to `1.1.1`. What is recommended for patch is to:
1. Branch off from the `0.8.3` branch
2. Revert the current master to `0.8.3` and merge the PR to master
3. Revert back the current master to `1.1.1` after the above is done.